scripts/benchmark_multithreaded.py

```diff
@@ -0,0 +1,190 @@
+# To prevent _tkinter.TclError: https://stackoverflow.com/a/37605654
+import matplotlib
+matplotlib.use('Agg')
+import matplotlib.pyplot as plt
+
+import os
+import argparse
+import multiprocessing
+import subprocess
+import json
+
+from datetime import datetime
+
+
+
+BENCHMARK_EXECUTABLE = 'hyriseBenchmarkTPCH'
+DEFAULT_TPCH_QUERIES = [query for query in range(1, 23) if query != 15] # Exclude query 15 which is not supported in our multithreaded benchmarks
+MAX_CORE_COUNT = multiprocessing.cpu_count()
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--scale', action='store', type=float, metavar='S', default=0.1, help='TPC-H scale factor (default: 0.1)')
+    parser.add_argument('-v', '--verbose', action='store_true', help='Print log messages')
+    parser.add_argument('-q', '--queries', action='store', type=int, metavar='Q', nargs='+', help='Specify the TPC-H queries that will be benchmarked')
+    parser.add_argument('--chunk-size', action='store', type=int, metavar='S', help='Specify maximum chunk size (default: Maximum available)')
+    parser.add_argument('--result-dir', action='store', type=str, metavar='DIR', default='results', help='Directory where the result folder will be stored (default: \'results/\')')
+    parser.add_argument('--result-name', action='store', type=str, metavar='NAME', help='Directory where the actual results will be stored (default: current datetime)')
+    parser.add_argument('--build-dir', action='store', type=str, metavar='DIR', required=True, help='Directory that contains the hyriseBenchmarkTPCH executable')
```

Comment:
Why do you take the build directory instead of the binary? How about this syntax, which allows us to pass any type of arguments to the binary without explicitly listing them in this script:

`benchmark_multithreaded.py [--script-options=foo] ./build-release/hyriseBenchmarkTPCH [--hyrise_options]`